demos/config.h

```diff
@@ -22,9 +22,37 @@
 #ifndef CONFIG_H
 #define CONFIG_H
 
+
 /* Set network context to socket (int). */
 typedef int MQTTNetworkContext_t;
 
 #define MQTT_MAX_QUEUED_PUBLISH_MESSAGES 10
 
+/*
+ * @brief The Logging Interface Hook.
```

Comment:
The documentation about ```IotLog``` macro needs to be placed in the ```iot_log_setup.h``` file, informing the user that they need to define the following macro to map to their platform's logging function:

```c
#define IotLog( messageLevel, pFormat, ... )
```